projects/js-packages/publicize-components/src/components/panel/index.jsx

```diff
@@ -59,7 +59,7 @@ const PublicizePanel = ( { prePublish, children } ) => {
 									  )
 							}
 							onChange={ togglePublicizeFeature }
-							checked={ isPublicizeEnabled }
+							checked={ isPublicizeEnabled && hasConnections }
```

Comment:
Do we change this so the toggle is in the off state while disabled? The module is technically enabled so I'm not sure if it makes it less confusing

Reply:
> Do we change this so the toggle is in the off state while disabled?

Yes, otherwise it will be confusing that it's disabled but turned ON.